independent-publisher-connectors/NFTMania/MintNFT.swagger.json

```diff
@@ -0,0 +1,84 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "MintNFT",
+    "description": "NFT Mania provides you with an Ultimate NFT experience and lets you develop applications in no time!",
+    "version": "1.0"
+  },
+  "host": "api.nftport.xyz",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/v0/mints/easy/urls": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Easy minting with URL",
+        "description": "Easy minting with URL",
```

Comment:
@shreyan1999, all operations descriptions should end with a period, No period required only if there's a single word. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations)